src/promptflow/promptflow/executor/flow_validator.py

```diff
@@ -73,9 +75,11 @@ def _validate_nodes_topology(flow: Flow) -> Flow:
         for node in flow.nodes:
             if node.name in node_names:
                 raise DuplicateNodeName(
-                    message=f"Node with name '{node.name}' appears more than once in the node definitions in your "
-                    f"flow, which is not allowed. To address this issue, please review your "
-                    f"flow and either rename or remove nodes with identical names.",
+                    message_format="Flow is defined incorrectly. Node with name '{node_name}' appears more than "
+                    "once in the node definitions in your flow, which is not allowed. To address this "
+                    "issue, please review your flow and either rename or remove nodes "
```

Comment:
To split the long message into multiple lines, we can try to split it from a natural place (i.e. the ending of a sentence).

We might prefer:
```
(
    "Flow is defined incorrectly. Node with name '{node_name}' appears "
    "more than once in the node definitions in your flow, which is not allowed. "
    "To address this issue, please review your flow and either rename "
    "or remove nodes with identical names.",
)
```

over:
```
(
    "Flow is defined incorrectly. Node with name '{node_name}' appears more than "
    "once in the node definitions in your flow, which is not allowed. To address this "
    "issue, please review your flow and either rename or remove nodes "
    "with identical names.",
)
```